docs/manage/security/log-search-audit-index.md

```diff
@@ -53,8 +53,8 @@ The table below shows the possible values for the field, query_type.
 | View Maintenance | Scheduled View queries run on behalf of the users in the org. |
 | Sumo Internal | The Internal searches Sumo runs in the background that are critical in providing other services (For example, autocomplete, scheduled view optimization, etc.)                                    |
 | Live Dashboard | Search queries used to power live dashboard panels. |
-| Compare | Search queries that used the logreduce or compare timeshift operators generated two audit logs. Instead of generating two audit logs for the parent and child queries, a single audit log will now capture the scanned bytes for both queries and store them in a single audit index. As a result, the scanned bytes for Compare queries will be found as a sum of the parent and child queries in the parent query audit index. |
-| Subquery | Subqueries that were run as a separate search with corresponding attributes captured in the Search Audit Index generated two audit logs. Instead of generating two audit logs for the parent and child queries, a single audit log will now capture the scanned bytes for both queries and store them in a single audit index. As a result, the scanned bytes for Subquery queries will be found as a sum of the parent and child queries in the parent query audit index.    |
+| Compare | Search queries that used the logreduce or compare timeshift operators generated two audit logs. Instead of generating two audit logs for the parent and child queries, a single audit log will now capture the scanned bytes for both queries and store them in a single audit index. As a result, the scanned bytes for Compare queries will be found as a sum of the parent and child queries in the parent query audit index.  |
+| Subquery | Subqueries that were run as a separate search with corresponding attributes captured in the Search Audit Index generated two audit logs. Instead of generating two audit logs for the parent and child queries, a single audit log will now capture the scanned bytes for both queries and store them in a single audit index. As a result, the scanned bytes for Subquery queries will be found as a sum of the parent and child queries in the parent query audit index.   |
```

Comment:
I would reword this as the following: 

You will not be able to see "Subquery" kind of audit logs anymore. This was applicable for queries that ran the subquery operator, where apart from the main audit log, we generated a "Subquery" audit log showcasing the attributes of a child query. Now, a single audit log will capture the sum of scanned bytes for both parent and child queries.